### ISSUE TITLE
'Linkify Twitter Trends': the linkified trend text is blue (instead of black/white)

### ISSUE THREAD

**Issue by darkred** [chocolateboy/userscripts#12]
Thanks for another very useful script, 'Linkify Twitter Trends'!

I've noticed with the script, that the linkified trend text is blue (instead of black/white).
Could you please make it remain the same, i.e. black in Light(Default) and black in Dim/Lights out?
*The underline will still serve as a hint that it's a link.*

Screenshot comparison: 
<details>
<summary>without the script</summary>

![Light](https://user-images.githubusercontent.com/723651/91605835-c6921b00-e979-11ea-8b60-fc36343ea22c.jpg) ![Dim](https://user-images.githubusercontent.com/723651/91605838-c85bde80-e979-11ea-9d88-313504bf10c0.jpg)

</details>
<details>
<summary>with the script</summary>

![Light](https://user-images.githubusercontent.com/723651/91605991-08bb5c80-e97a-11ea-931d-37da4a1e2386.jpg) ![Dim](https://user-images.githubusercontent.com/723651/91605997-0b1db680-e97a-11ea-9aa2-261ddacb6df6.jpg)

</details>

---

PS. I'd like to ask you please to rename it to "Twitter Linkify Trends", i.e. have the "Twitter" word first, like your other two scripts twitter, so that it's properly listed (alphabetically) in TM/VM popup.

**Comment by chocolateboy** [comment]
Thanks for letting me know! Should be fixed in [v0.1.0](https://github.com/chocolateboy/userscripts/commit/08ce2553eb040980f57ba84b166e937dfdfc7b4d).

I removed the underline since Twitter doesn't underline links, and this is just meant to make trends behave the way they used to behave, and still *appear* to behave, rather than introducing a new look.

As for the name change, I like it, but I couldn't get it to work: Violentmonkey (and presumably the other engines) ends up with two copies of the same userscript with different names, which no-one wants, few would notice/remedy, and which would inevitably lead to conflicts, so sadly it's not to be. Sorry!

**Comment by darkred** [comment]
Thank you for the fix!

Regarding the underline: I still see it with v0.1.0 .
![2020-08-29_014944](https://user-images.githubusercontent.com/723651/91623108-c8240900-e9a2-11ea-855e-d9450633ad7e.jpg)
Maybe you removed it with a newer commit that you haven't pushed yet?

---

> I like it, but I couldn't get it to work:

Mo, it can be done and works ok!
I've just checked (with [this test script I posted](https://greasyfork.org/en/scripts/410128-test-5) ) in both Violentmonkey and Tampermonkey,
and if you change the name of the script, and bump the version, e.g. `Test 1` --> `Test 2` and version `0.1` --> `0.2`
only one script will be after the script update check: the renamed one.

**Comment by chocolateboy** [comment]
Thanks, the underline should be fixed in v0.3.0.

I tried renaming, but had to revert it as it doesn't work in Violentmonkey for Firefox (the update hangs). It works fine in Violentmonkey for Chrome.

**Comment by darkred** [comment]
> it doesn't work in Violentmonkey for Firefox (the update hangs). 

Are you sure?
I'd gladly make an issue for that in the Violentmonkey repo, 
but I just tested it in a new Firefox profile, and it works perfectly fine (I've tried several times):

STR
- Firefox 80 x64 (fresh profile), Violentmonkey 2.12.7 (no scripts installed), 
- I installed my `Test 1` script, changed the title (e.g. to `Test 4`) + bumped the version (to `0.4`),
 and then pressed either the global "Check all for updates" or the scripts "Check for updates":
the script gets updated ok, having the new title+version.
- Maybe you'd like to make to post a new e.g. "test" script in your Greasyfork account yourself, then change title+version, and I'm sure it will works fine for you too.
